capitalizedecorator.rb

```diff
@@ -0,0 +1,12 @@
+require './basedecorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def initialize(nameable)
+    super()

```

Comment:
- [ ] After implementing the changes requested by the previous reviewer, I'm sorry to say that your app is still not working as expected as you can see below 👇 
![image](https://user-images.githubusercontent.com/46839548/213427518-fc4b273c-2d19-4bb5-9ceb-8a5043dfcfa4.png)
Please remove the `super()` line of code from your `basedecorator.rb`, `capitalizedecorator.rb` and `trimmerdecorator.rb` files and your app will run just fine 😎  like this 👇  
![image](https://user-images.githubusercontent.com/46839548/213427324-b46df4aa-2dff-459a-ab44-a95f30d7ab46.png)